CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changes
 =======
 
+0.4.0 (YYYY-MM-DD)
+------------------
+
+* Products outside of the domain can now be crawled using
+  ``zyte_spider_templates.middlewares.AllowOffsiteMiddleware``.
+
+* Updated documentation to also setup ``zyte_common_items.ZyteItemAdapter``.
+
+* Improved ``max_requests`` spider parameter description.
```

Comment:
```suggestion
* Products outside of the target domain can now be crawled using
  :class:`zyte_spider_templates.middlewares.AllowOffsiteMiddleware`.

* Updated the documentation to also set up ``zyte_common_items.ZyteItemAdapter``.

* Improved the description of the ``max_requests`` spider parameter.
```

I am not 100% sure of whether or not `:class:` will make the doc build fail if the class is not in the docs, so run `tox -e docs` to make sure, and revert if it causes the build to fail. Or add the class somewhere in the docs :slightly_smiling_face:.

Reply:
It does work! 🙌